atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/TimestampMapping.java

```diff
@@ -31,7 +35,7 @@ interface TimestampMapping {
 
     @Value.Derived
     default Range<Long> versionRange() {
-        LongSummaryStatistics summary = timestampMapping().values().stream()
+        LongSummaryStatistics summary = checkNotEmpty(timestampMapping()).values().stream()
```

Comment:
I believe `checkNotEmpty` shouldn't be necessary given the `nonEmptyMapping` check?

Reply:
Good call, will remove